osclib/stagingapi.py

```diff
@@ -1456,6 +1468,27 @@ def create_adi_project(self, name, use_frozenlinks=False, src_prj=None):
         # put twice because on first put, the API adds useless maintainer
         http_PUT(url, data=meta)
 
+        prjconf = textwrap.dedent("""
+            # prjconf for testing reproducible builds:
+            BuildFlags: nodisturl
+            Release: 1.1
+
+            %if %_repository == "rb_future1y"
+            VMinstall: reproducible-faketools-futurepost
```

Comment:
should probably be 'Required'` - not `VMInstall`

Reply:
or even `support` - i.e do not trigger rebuilds down the line just becuase those packages might have changes